src/include/ndpi_protocol_ids.h

```diff
@@ -361,6 +361,7 @@ typedef enum {
   NDPI_PROTOCOL_ELASTICSEARCH         = 330,
   NDPI_PROTOCOL_TUYA_LP               = 331, /* TUYA LAN Protocol; IoT OS: https://github.com/tuya/tuya-iotos-embeded-sdk-wifi-ble-bk7231n */
   NDPI_PROTOCOL_TPLINK_SHP            = 332, /* TP-LINK Smart Home Protocol */
+  NDPI_PROTOCOL_THRIFT                = 333, /* Apache Thrift */
```

Comment:
Isn't there a `*_FREE_*` protocol you can use? (Sorry, can not verify at the moment as I have just my mobile available.)

Reply:
+1 on using a `*_FREE_*` id